src/html/runnabear.html

```diff
@@ -7,105 +7,104 @@ <h2 class="h2 weight-strong">Full Image</h2>
     <figure class="grid-block figure vertical shrink align-center justify-right figure-bears">
       <img class="img" src="images/bears/basic-bear.png" width="180" alt="Basic Bear">
       <figcaption class="figcaption">Basic Bear</figcaption>
-      <a class="btn btn-xs white" href="images/bears/basic-bear.png" download>Download</a>
-    </figure>
+      <a class="btn btn-xs white" href="images/bears/RGB/basic-bear.png" Download>Download PNG</a>
```

Comment:
Looks like this first figure is missing its closing tag:
![image](https://cloud.githubusercontent.com/assets/7440805/25600180/28ac81a0-2e97-11e7-90c6-6e23aab9c79f.png)